src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/TagletWriterImpl.java

```diff
@@ -531,6 +532,16 @@ public Content valueTagOutput(VariableElement field, String constantVal, boolean
                 : Text.of(constantVal);
     }
 
+    @Override
+    protected Content invalidTagOutput(String summary, Optional<String> detail) {
+        if (detail.isEmpty() || detail.get().isBlank()) {
+            return HtmlTree.SPAN(HtmlStyle.invalidTag, Text.of(summary));
+        }
+        return new HtmlTree(TagName.DETAILS).addStyle(HtmlStyle.invalidTag)
```

Comment:
This 3-state detail bothers me:

  1. The detail is absent (i.e. the optional is empty)
  2. The detail is a blank
  3. The detail is non-blank

If we care about a blank detail (i.e. an error description that consists only of whitespace), then why treat it like an absent detail and omit it from the documentation? If we don't care about a blank detail, then why wrap it in an `Optional` when a blank string would do?

Reply:
> it's just that it's silly to display a summary/detail box unless you have something to show in the details.

Agreed. This is just another case of whether it is better to assert/check that the input is sensible, or to silently do something reasonable in the face of unreasonable arguments.